File: MPI/mpicheck/tests/functions/1.0/MPI_Get_processor_name.f

        
        program main
        include 'mpif.h'
        
        CHARACTER*(10) var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_get_processor_name(var_0, var_1, var_2)
       call pmpi_get_processor_name(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Get_processor_name.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    char *var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Get_processor_name(var_0, var_1);
    ret = PMPI_Get_processor_name(var_0, var_1);
    return 0;
}
